test/extended/openshift/extended_test.go

```diff
@@ -109,7 +109,7 @@ func TestBasic(t *testing.T) {
 	}
 	defer w.Stop()
 
-	timeout := 60 * time.Second
+	timeout := 600 * time.Second
```

Comment:
60s should be fine; this is too much. this timeout in the test means it failed to receive cert